packaging/installer/methods/manual.md

```diff
@@ -6,217 +6,88 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/instal
-
-# Install Judy-Devel directly
-yum install -y http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/Judy-devel-1.0.5-18.module_el8.1.0+217+4d875839.x86_64.rpm
+packaging/installer/install-required-packages.sh netdata
 ```
 
 ## Install Netdata
 
-Do this to install and run Netdata:
+We provide a script that encapsulates the entire build and install process as a single command. You can run it
+from the root of your checkout of the Netdata git repository like so:
 
 ```sh
-# download it - the directory 'netdata' will be created
-git clone https://github.com/netdata/netdata.git --depth=100 --recursive
-cd netdata
-
-# run script with root privileges to build, install, start Netdata
 ./netdata-installer.sh
 ```
 
 -   If you don't want to run it straight-away, add `--dont-start-it` option.
-
--   You can also append `--stable-channel` to fetch and install only the official releases from GitHub, instead of the nightly builds.
-
--   If you don't want to install it on the default directories, you can run the installer like this: `./netdata-installer.sh --install /opt`. This one will install Netdata in `/opt/netdata`.
-
--   If your server does not have access to the internet and you have manually put the installation directory on your server, you will need to pass the option `--disable-go` to the installer. The option will prevent the installer from attempting to download and install `go.d.plugin`. 
+-   If you don't want to install it on the default directories, you can run the installer like this:
+    `./netdata-installer.sh --install /opt`. This one will install Netdata in `/opt/netdata`.
 
 ## Optional parameters to alter your installation
 
 `netdata-installer.sh` accepts a few parameters to customize your installation:
 
--   `--dont-wait`: Enable automated installs by not prompting for permission to install any required packages.
+-   `--dont-wait`: Run without prompting for any user input.
 -   `--dont-start-it`: Prevent the installer from starting Netdata automatically.
--   `--stable-channel`: Automatically update only on the release of new major versions.
--   `--nightly-channel`: Automatically update on every new nightly build.
 -   `--disable-telemetry`: Opt-out of [anonymous statistics](/docs/anonymous-statistics.md) we use to make
     Netdata better.
--   `--no-updates`: Prevent automatic updates of any kind.
--   `--reinstall`: If an existing install is detected, reinstall instead of trying to update it. Note that this
-    cannot be used to change installation types.
--   `--local-files`: Used for [offline installations](offline.md). Pass four file paths: the Netdata
-    tarball, the checksum file, the go.d plugin tarball, and the go.d plugin config tarball, to force kickstart run the
-    process using those files. This option conflicts with the `--stable-channel` option. If you set this _and_
-    `--stable-channel`, Netdata will use the local files.
 
 ### Connect node to Netdata Cloud during installation
 
```

Comment:
Making a comment here bc Github prevents me to add comments on unchanged parts of the docs: 

I would include a heading before line 99: "Known issues"

Reply:
Agreed.